rpc/wrtc_call_queue_mongodb.go

```diff
@@ -614,10 +625,12 @@ func (queue *mongoDBWebRTCCallQueue) processNextSubscriptionEvent(next mongoutil
 				// though, we want to send the events as fast as possible as mentioned
 				// above and cannot block on the send to see if the receiver locked
 				// the document.
+				queue.logger.Infow("sending event", "event", event)
```

Comment:
[q] wouldn't this log once per channel without including identifying information for the channel? Would it make sense to instead log the event in the log line above at line 619?

(Apologies if I'm misunderstanding here!)

Reply:
I actually just think we should remove this here, it will be noisy and it was mainly for me debugging locally :)